lib/dom-actions.ts

```diff
@@ -0,0 +1,152 @@
+import { ElementSelector, HideMethod, VisibilityCheck } from "./rules";
+import { DomActionsProvider } from "./types";
+import { getStyleElement, hideElements, isElementVisible, waitFor } from "./utils";
+import AutoConsent from "./web";
+
+export class DomActions implements DomActionsProvider {
+  constructor(public autoconsentInstance: AutoConsent) { }
+
+  click(selector: ElementSelector, all = false): boolean {
+    const elem = this.elementSelector(selector)
+    this.autoconsentInstance.config.logs.rulesteps && console.log("[click]", selector, all, elem);
+    if (elem.length > 0) {
+      if (all) {
+        elem.forEach((e) => e.click());
+      } else {
+        elem[0].click();
+      }
+    }
+    return elem.length > 0;
+  }
+
+  elementExists(selector: ElementSelector): boolean {
+    const exists = this.elementSelector(selector).length > 0;
+    // enableLogs && console.log("[exists?]", selector, exists);
```

Comment:
These log messages are not used anymore, and replaced by a log up the stack, right? Can we just remove them instead of commenting?

Reply:
yep, this is just a copypaste from the old file, I'll remove these lines